Tests/BuiltInRulesTests/LegacyObjcTypeRuleTests.swift

```diff
@@ -0,0 +1,20 @@
+@testable import SwiftLintBuiltInRules
+import TestHelpers
+import XCTest
+
+final class LegacyObjcTypeRuleTests: SwiftLintTestCase {
```

Comment:
We prefer test cases to be part of the triggering and/or non-triggering example in the rule's description if possible. `Example` allows you to add a `configuration` argument if options influence the outcome. In other words, this dedicated test class isn't needed.

The advantage is that these examples also serve as documentation.

Reply:
Thank you for review @SimplyDanny . I've made changes.